doc/_static/theme_overrides.css

```diff
@@ -0,0 +1,14 @@
+/* https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html */
+/* override table width restrictions */
```

Comment:
This is to fix the table. Without this, the long sentences in tables will overflow with horizontal scroll instead of wrap properly.